packages/integrations/netlify/src/netlify-functions.ts

```diff
@@ -80,6 +80,9 @@ export const createExports = (manifest: SSRManifest, args: Args) => {
 				locals = JSON.parse(localsAsString);
 			}
 		}
+		if (builders) {
+			Object.assign(locals, { netlify: { builders: { ttl : undefined } } });
```

Comment:
I don't think there's a specific need to create such a deep object for **one value**. What about 

```js
{
	netlifyBuildersTtl: 45
}
```

Reply:
I am concerned that this is prone to deprecation. For example, what if we introduce more runtime-specific controls that would benefit from a namespace.

Reply:
I would say that's not a concern because all these objects are controlled by the adapter, and we can have only one adapter at a time. 

Maybe `netlify.buildersTtl` should be enough.

Reply:
I agree, going with `netlify.setBuildersTtl()`.

Reply:
Implemented